app/views/PageNotFoundView.scala.html

```diff
@@ -0,0 +1,33 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+        layout: templates.Layout,
+        govukButton: GovukButton
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = titleNoForm(messages("pageNotFound.title"))) {
+
+    <h1 class="govuk-heading-xl">@messages("pageNotFound.h1")</h1>
+    <p class="govuk-body">@messages("pageNotFound.p1")</p>
+    <p class="govuk-body">@messages("pageNotFound.p2")</p>
+    <p class="govuk-body">@messages("pageNotFound.p3")
+    <p class="govuk-body"><a href=" @routes.HelpAndSupportController.onPageLoad()" target="_blank" class="govuk-link">@messages("pageNotFound.p3.linkText")</a> @messages("pageNotFound.p3.1")</p>
```

Comment:
This isn't formatting the page content right, the link shouldn't be on a separate link.